ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -308,3 +308,45 @@ Verify container images
     Container Image Url Should Contain      ${APPLICATIONS_NAMESPACE}     ${pod}      ${container}
     ...     ${registry_name}/rhoai/${expected_image}
     Log To Console    ${pod_name}'s container image is verified
+
+Enable appwrapper in Codeflare operator
+    [Documentation]    Enable appwrapper in Codeflare operator
+    ${result} =    Run Process    oc -n ${APPLICATIONS_NAMESPACE} get configmap codeflare-operator-config -o yaml | sed 's/enabled: false/enabled: true/' | oc apply -n redhat-ods-applications -f -
```

Comment:
I guess this also overrides kueueJobReconciller/waitForPodsReady , is it so?
Would it be possible to use yq to modify just the specific field?

Reply:
Fixed, now it is using yq.